patches/tModLoader/Terraria/release_extras/DedicatedServerUtils/manage-tModLoaderServer.sh

```diff
@@ -15,6 +15,20 @@ function popd {
 	command popd > /dev/null || return
 }
 
+function try_make_dir {
+	for dir in "$@"; do
+		if ! [[ -d "$dir" ]]; then
+			mkdir "$dir"
+		fi
+	done
```

Comment:
is `mkdir -p` not acceptable for some reason?

Reply:
I didn't realize `-p` removes the errors as well, weird behavior but I'll convert all of them to use it. Thanks!